remote/src/main/java/com/example/appengine/remote/RemoteApiSeedPageViewData.java

```diff
@@ -0,0 +1,215 @@
+package com.example.appengine.remote;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.GeoPt;
+import com.google.appengine.tools.remoteapi.RemoteApiInstaller;
+import com.google.appengine.tools.remoteapi.RemoteApiOptions;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class RemoteApiSeedPageViewData {
+  public static void main(String[] args) throws IOException {
+    String serverString = args[0];
+    System.out.println(serverString);
+    RemoteApiOptions options;
+    if (serverString.equals("localhost")) {
+      options = new RemoteApiOptions().server(serverString, 8080).useDevelopmentServerCredential();
+    } else {
+      options = new RemoteApiOptions().server(serverString, 443).useApplicationDefaultCredential();
+    }
+    RemoteApiInstaller installer = new RemoteApiInstaller();
+    installer.install(options);
+    try {
+      DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+      Entity pageViews1 = new Entity("PageViews");
+      pageViews1.setProperty("restaurantKey", 1);
```

Comment:
So in the end are we still going to use the restaurantKey? To my understanding we were changing to just use the key created by the entity itself.

Reply:
I thought we were still putting restaurantKeys in the PageViews entities? They won't be in the RestaurantInfo entities anymore but do we need them for PageViews?

Reply:
Come to think of it, I don't think I need the keys for my visualization, so if we want to take that out I'm happy to. I think the MetricsClient would need to be updated but let me know what you think.

Reply:
I'm not sure, but from my understanding, the only place that the entity key would be used is in the `RestaurantInfo` entity. Every other entity kind would just use the ID value extracted from the key of the `RestaurantInfo` entity.

Reply:
That way, everything else would stay exactly as it was.

Reply:
Ideally there should only be 1 id/key used to uniquely identify the object. Since datastore generates a unique ID automatically, we should use that. However, I understand that we've already used the restaurant key string in many places, so if this would require a lot of work to change we can put in a todo and fix it later if there is time.